config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
 
+  get 'social_profiles/index'
```

Comment:
I removed these - didn't catch that they were created. I think it was when I created the Social Profile controller.